src/lib/containers/row_renderers/Study.tsx

```diff
@@ -31,18 +31,19 @@ export default class Study extends React.Component<StudyProps, StudyState> {
 
     render() {
         const { data, schema } = this.props;
-        const projectName = data[schema.projectName];
-        const projectLeads = data[schema.projectLeads] && data[schema.projectLeads].split(";").join(" / ");
-        let summary = data[schema.summary];
-        const diseaseFocus = data[schema.diseaseFocus];
-        const tumorType = data[schema.tumorType];
-        const projectStatus = data[schema.projectStatus];
-        const fundingAgency = data[schema.fundingAgency];
-        const dataStatus = data[schema.dataStatus];
-        const summarySource = data[schema.summarySource];
+        const projectName: string = data[schema.projectName];
```

Comment:
We should push for projectName to be renamed to name (in the future, if this is built on top of a view then "name" is already a default column that is surfaced for each entity type).